pallets/time-release/src/benchmarking.rs

```diff
@@ -16,8 +16,8 @@ pub type Schedule<T> = ReleaseSchedule<BlockNumberFor<T>, BalanceOf<T>>;
 const SEED: u32 = 0;
 
 fn set_balance<T: Config>(who: &T::AccountId, balance: BalanceOf<T>) {
-	let actual_deposit = T::Currency::mint_into(&who, balance.saturated_into());
-	assert_eq!(balance, actual_deposit.unwrap());
+	let actual_deposit = T::Currency::set_balance(&who, balance.saturated_into());
```

Comment:
`mint_into` fails when the account has a `0` balance. 
`set_balance` is infallible and provided by polkadot-sdk specifically for benchmarking and testing.